lib/tasks/census.rake

```diff
@@ -1,8 +1,8 @@
 namespace :census do
   desc "Update cohort on all users with census data"
-  task update_cohorts: :environment do
+  task update: :environment do
     User.all.each do |user|
-      user.add_cohort
+      user.add_cohort_and_role
```

Comment:
@Dpalazzari Yea I think so. I was hoping to run it with the update for the token so was going to wait to make a combined rake task that ran both and schedule that one.